### ISSUE TITLE
Does it support export to onnx?

### ISSUE THREAD

**Issue by jinfagang** [as-ideas/TransformerTTS#8]
Can it support export to onnx and inference using other language or paltforms?

**Comment by cfrancesco** [comment]
I did not try yet, sorry. In case you do, would be great if you reported here.

**Comment by jinfagang** [comment]
Yeah sure, just wonder tensorflow is onnx the best to deploy model.
